### PR TITLE
Setup autodoc

### PR DESCRIPTION
**Pull request by pydanny**
- Loaded Django into Sphinx so autodoc would work
- Added more directory levels to the path so autodoc can discover Python modules
- Added autodoc for all modules

Note: The Sphinx theme we're using isn't the best for autodoc, we'll move to a new theme in a separate PR.

### CODE REVIEW

**Review by pydanny — COMMENTED** [review]
Ugh, tried to resolve a merge conflict quickly and created a merge commit 🤦‍♂️

**Comment by jbkkd** [comment]
Just do an interactive rebase and the merge commit will disappear

**Comment by pydanny** [comment]
Heh, I'm so out of practice with team-based git usage. I keep compounding my errors. 🔥

**Comment by pydanny** [comment]
Okay, commit issue is fixed now. It will be good to get my git-fu back up.

File: docs/conf.py [hunk 1]
```diff
@@ -10,15 +10,16 @@
...
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
 
+import django
 
-# TODO: Use with autodoc
-# from django.conf import settings
+sys.path.insert(0, os.path.abspath(".."))
```
Comment (jbkkd) [inline docs/conf.py:18]: Could use a comment on why we're inserting both this folder and the one above it to the path

**Review by jbkkd — APPROVED** [review]
Can relate - I've been in the world of "How do I code?" recently.

PR looks good!

**Pull request merged**